app/classes/Transvision/ShowResults.php

```diff
@@ -270,18 +268,18 @@ public static function resultsTable($search_id, $search_results, $recherche, $lo
                      </thead>
                      <tbody>\n";
 
-        if (!$search_options['whole_word'] && !$search_options['perfect_match']) {
-            $recherche = Utils::uniqueWords($recherche);
+        if (! $search_object->isWholeWords() && ! $search_object->isPerfectMatch()) {
+            $recherche = Utils::uniqueWords($search_object->getSearchTerms());
```

Comment:
Since we're touching this piece of code, can we switch to the English name for the variable? ;-)